### PR TITLE
Version 2.5.0

### PR DESCRIPTION
**Pull request by pizzaboxer**

### CODE REVIEW

**Comment by bluepilledgreat** [comment]
Looks bad to me.

**Comment by pizzaboxer** [comment]
woah thats sick i didnt know prs can auto close issues

**Review by Sahelantrophus — COMMENTED** [review]
looks good

File: Bloxstrap/FastFlagManager.cs [hunk 1]
```diff
@@ -207,8 +203,24 @@ public override void Load()
         {
             base.Load();
 
-            SetPresetOnce("Rendering.Framerate", 9999);
             CheckManualFullscreenPreset();
+
+            if (GetPreset("Rendering.Framerate") is not null)
+                return;
+
+            // set it to be the framerate of the primary display by default
```
Comment (Sahelantrophus) [inline Bloxstrap/FastFlagManager.cs:211]: Hi nephew

**Pull request merged**